tbx/project_styleguide/templates/patterns/organisms/header/header.html

```diff
@@ -1,65 +1,74 @@
+                    Home
+                    {% include "patterns/atoms/icons/icon.html" with name="chevron" classname="header__primary-menu-toggle-icon" %}
+                </button>
 
             {# Primary mobile navigation #}
-            <nav aria-label="Main navigation"  class="primary-nav-mobile" data-primary-mobile-menu>
-                {% wagtailcache 600 "primarynavmobile" current_site.pk is_pattern_library %}
-                    <ul class="primary-nav-mobile__list">
-                        <li class="primary-nav-mobile__item">
-                            <a href="/" aria-label="Homepage link" class="primary-nav-mobile__link">
-                                <span class="primary-nav-mobile__text">Home</span>
-                            </a>
-                        </li>
-                        {% primarynavmobile %}
-                    </ul>
-                {% endwagtailcache %}
-            </nav>
+                <nav aria-label="Main navigation"  class="primary-nav-mobile" data-primary-mobile-menu>
+                    {% wagtailcache 600 "primarynavmobile" current_site.pk is_pattern_library %}
+                        <ul class="primary-nav-mobile__list">
+                            <li class="primary-nav-mobile__item">
+                                <a href="/" aria-label="Homepage link" class="primary-nav-mobile__link">
+                                    <span class="primary-nav-mobile__text">Home</span>
+                                </a>
+                            </li>
+                            {% primarynavmobile %}
+                        </ul>
+                    {% endwagtailcache %}
+                </nav>
 
-            {# Mode switcher desktop - hide for now #}
-            {% comment %} {% include "patterns/molecules/mode_switcher/mode_switcher.html" %} {% endcomment %}
+                {% include "patterns/molecules/mode_switcher/mode_switcher.html" %}
+            </div>
         </div>
-        {# Mobile menu toggle -  hide for now, will be reused in the secondary nav #}
-        {% comment %} <div class="header__menu-toggle grid__header-toggle">
-            <span class="sr-only">Menu</span>
-            <button class="button-menu-toggle" data-mobile-menu-toggle aria-haspopup="true" aria-expanded="false" aria-label="Mobile menu toggle">
-                <span class="button-menu-toggle__line"></span>
-                <span class="button-menu-toggle__line"></span>
-                <span class="button-menu-toggle__line"></span>
-                <span class="button-menu-toggle__line"></span>
-            </button>
+    </div>
+    <div class="grid">
+        <div class="header__bottom grid__header-nav">
+            <div class="header__logo">
+                {% include "patterns/atoms/icons/icon.html" with name="logo" classname="logo" %}
+                <span class="logo__label">
```

Comment:
Q: Does this need to be wrapped in `{% if page.label or page.final_division and page.final_division.label %}` so we don't render an empty span if all are false? Not sure how common that scenario would be 🤔

Reply:
great point! definitely needed to add this since we're not going to need the secondary nav on the new home page.